game/magic/game/game.go

```diff
@@ -6002,6 +6003,21 @@ func (game *Game) doGaiasBlessing(player *playerlib.Player) {
                 }
             }
         }
+
+        // lose city population due to pestilence
+        if city.HasEnchantment(data.CityEnchantmentPestilence) {
+            if city.Citizens() >= 11 || city.Citizens() > (rand.IntN(10) + 1) {
+                city.Population -= 1000
```

Comment:
I'm not quite sure about this part removing 1000 population, does this make sense?